benchmarks/active_record/benchmark.rb

```diff
@@ -0,0 +1,33 @@
+require "harness"
+require "bundler/inline"
+
+gemfile do
+  source "https://rubygems.org"
+  gem "activerecord", "~> 6.0.3", ">= 6.0.3.6"
+  gem "sqlite3", "~> 1.4"
+end
+
+require "active_record"
+
+ActiveRecord::Base.establish_connection adapter: "sqlite3", database: ":memory:"
+
+ActiveRecord::Schema.define do
+  create_table :posts, force: true do |t|
+    t.string :title
+    t.string :body
+  end
+end
+
+class Post < ActiveRecord::Base; end
+post = nil
+50000.times { post = Post.create!(title: "hi", body: "hello!") }
```

Comment:
Let's bump that to 200K

Reply:
Does that matter?  We're not fetching all 50k objects in the benchmark.

Reply:
Yes that's a good point. Let's keep your 50K record and fetch more instead, like 1000 or 2000. That will benchmark Ruby more heavily, which is what we want.